src/main/kotlin/frc/robot/subsystems/gripper/GripperIOSim.kt

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.gripper
+
+import com.ctre.phoenix6.controls.DutyCycleOut
+import edu.wpi.first.units.Units
+import edu.wpi.first.wpilibj.Timer
+import frc.robot.lib.motors.TalonFXSim
+
+class GripperIOSim : GripperIO {
+    override val inputs = LoggedGripperInputs()
+    private val rollerMotor = TalonFXSim(1, 1.0, 0.5, 1.0)
```

Comment:
If the real IO uses a Spark MAX, there is no reason to simulate it with a Talon FX.

Reply:
e31422fbd6373b3bbbf2559e9f6083674d91da43